src/main/java/com/fasterxml/jackson/databind/deser/std/SqlBlobDeserializer.java

```diff
@@ -0,0 +1,54 @@
+package com.fasterxml.jackson.databind.deser.std;
+
+import java.io.IOException;
+import java.sql.Blob;
+import java.util.Arrays;
+import java.util.UUID;
+
+import javax.sql.rowset.serial.SerialBlob;
```

Comment:
Hmmmh. I wonder... this seems to require dependency to another JDK9+ module

https://docs.oracle.com/en/java/javase/11/docs/api/java.sql.rowset/javax/sql/rowset/serial/package-summary.html

beyond basic `java.sql`. Will need to add to Moditect sources I think